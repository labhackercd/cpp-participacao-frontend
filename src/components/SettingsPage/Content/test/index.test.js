import React from 'react';
import ReactDOM from 'react-dom';
import MockTheme from '../../../mockTheme'

import {shallow, render, mount} from "enzyme";
import SettingsContent from './../index';

it("snapshot should not have changes", () => {
    const component = shallow(<SettingsContent/>);
    expect(component.exists()).toEqual(true);
    expect(component).toMatchSnapshot();
});

test('Test if NavBar renders without crash', () => {
    const div = document.createElement("div")
    ReactDOM.render(<SettingsContent/>, div)
    ReactDOM.unmountComponentAtNode(div)
});

