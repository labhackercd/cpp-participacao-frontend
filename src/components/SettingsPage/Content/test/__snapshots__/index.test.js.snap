// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot should not have changes 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-content-3"
>
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-homeMenuContainer-1"
    container={true}
    spacing={3}
  >
    <HeaderSection />
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={12}
    >
      <WithStyles(ForwardRef(Grid))
        container={true}
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(Typography))
            color="textPrimary"
            gutterBottom={true}
          >
            Gerais
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(Paper))
            className="makeStyles-paper-2"
            style={
              Object {
                "minHeight": "20rem",
              }
            }
          >
            <GeneralListItem />
          </WithStyles(ForwardRef(Paper))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <ToolsSection />
    <AboutSection />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>
`;
